sentences/cs/_common.yaml

```diff
@@ -7,6 +7,58 @@ responses:
     no_device_class: "{{ area }} neobsahuje {{ device_class }}"
     no_entity: "Žádné zařízení ani entita se nejmenuje {{ entity }}"
     handle_error: "Při zpracování došlo k neočekávané chybě."
-lists: {}
-expansion_rules: {}
-skip_words: []
+lists:
+  color:
+    values:
+      - in: "bíl(á|é|ý|ou)"
+        out: "white"
+      - in: "čern(á|é|ý|ou)"
+        out: "black"
+      - in: "červen(á|é|ý|ou)"
+        out: "red"
+      - in: "oranžov(á|é|ý|ou)"
+        out: "orange"
+      - in: "žlut(á|é|ý|ou)"
+        out: "yellow"
+      - in: "zelen(á|é|ý|ou)"
+        out: "green"
+      - in: "modr(á|é|ý|ou)"
+        out: "blue"
+      - in: "fialov(á|é|ý|ou)"
+        out: "purple"
+      - in: "hněd(á|é|ý|ou)"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "Celsius"
+      - "fahrenheit"
+      - in: "f"
+        out: "Fahrenheit"
+expansion_rules:
+  name: "{name}"
+  area: "{area}"
+  what_is: "jak(á|é|ý) je"
+  how_many: "kolik je"
+  brightness: "{brightness} [percent]"
+  temp: "(teplota)"
+  temperature: "{temperature} [stupňů] [{temperature_unit}]"
+  turn_on: "zapni"
+  turn_off: "vypni"
+  turn_on_light: "(rozsviť|rozsviť)"
```

Comment:
two same values here

Reply:
orry, overlooked that.

Should be:

turn_off: "(vypni|vypnout|zhasni|zhasnout)"
turn_on_light: "(zapni|zapnout|rozsviť|rozsvítit)"

As I told, I don' know how to update PR patch from.